mseetc/ms/ocp.py

```diff
@@ -2,11 +2,11 @@
 import pandas as pd
 import casadi as ca
 
-from train import *
+from ..train import *
```

Comment:
I would - for now - undo the nested `ms` folder and update the imports (`from .track` instead of `from ..track`)